postcss.config.js

```diff
@@ -1,5 +1,19 @@
 module.exports = {
   plugins: [
+    [
+      "@fullhuman/postcss-purgecss",
+      {
+        contentFunction: (src) => {
+          if (src.endsWith("src/popup/index.scss")) {
+            return ["src/popup/**/*.{html,ts,tsx}"];
+          }
+          if (src.endsWith("src/options/index.scss")) {
+            return ["src/options/**/*.{html,ts,tsx}"];
+          }
+          return ["src/**/*.{html,ts,tsx}"];
```

Comment:
Maybe we should throw here for unexpected input source files?

Reply:
You mean e.g. someone sneeaking in a `.jsx` or `.svg` file? Is this the first and most obvious place where this would hit?

Reply:
Ah, thanks for asking. I rather meant if somebody creates a  new `index.scss` file in addition to the ones in `src/popup` and `src/options`. At the moment, we just fall back to scanning the whole `src/` directory for selectors which is sub-optimal and could easily go unnoticed.